assets/dev/js/editor/container/container.js

```diff
@@ -276,11 +276,11 @@ export default class Container extends ArgsObject {
 			return this;
 		}
 
-		if ( this !== this.renderer && this.renderer.view.isDestroyed ) {
+		if ( this !== this.renderer && this.renderer.view?.isDisconnected && this.renderer.view.isDisconnected() ) {
```

Comment:
`this !== this.renderer && this.renderer.view.isDisconnected()`?

Reply:
Not enough... causes error.